### DISCUSSION TITLE
请问在ubuntu下打包能够有参数设置不输出appimage包格式？

### DISCUSSION THREAD

**Discussion by sikenali** [tw93/Pake#354] [Q&A]
RT 仅需要打包成deb

**Comment by sikenali** [comment]
是不是去掉 tauri.linux.conf.json 下的appimage 参数即可？
不知道还是有其他操作或者打包会覆盖json配置吗

**Reply by Tlntin** [reply]
`去掉 tauri.linux.conf.json 下的appimage 参数即可`是的。

**Reply by sikenali** [reply]
> `去掉 tauri.linux.conf.json 下的appimage 参数即可`是的。

`{"tauri":{"bundle":{"icon":["/opt/QQ/resources/app/512x512.png"],"identifier":"pake-578345","active":true,"category":"DeveloperTool","copyright":"","deb":{"depends":["libwebkit2gtk-4.0-dev","build-essential","curl","wget","libssl-dev","libgtk-3-dev","libayatana-appindicator3-dev","librsvg2-dev","gnome-video-effects","gnome-video-effects-extra"]},"externalBin":[],"longDescription":"","resources":[],"shortDescription":"","targets":["deb"]}}}
`

你好去掉后，pake生成时又json被恢复了还是生成了appimage

**Reply by Tlntin** [reply]
懂你的意思了，我以为你是源码打包。用pake-cli的话，需要等我改改，增加一个可选参数来选择是输出deb，还是appimage还是两个都要。

**Reply by sikenali** [reply]
> 懂你的意思了，我以为你是源码打包。用pake-cli的话，需要等我改改，增加一个可选参数来选择是输出deb，还是appimage还是两个都要。

好的，感谢~
`ps 可能是我没有说清楚用的命令行`

**Reply by Tlntin** [reply]
pake-cli 1.2.9版增加了--targets参数，可选deb/appimage/all，默认为'all'，仅用于linux平台。
输入示例：
```bash
--targets xxx
```
对应的github actions也同步更新了表单参数，github actions表单默认输出格式是deb。
![image](https://user-images.githubusercontent.com/28218658/226083809-fb78a43e-16ac-4d1c-9c2f-01d17adb06ae.png)

**Reply by sikenali** [reply]
> pake-cli 1.2.9版增加了--targets参数，可选deb/appimage/all，默认为'all'，仅用于linux平台。 输入示例：
> 
> ```shell
> --targets xxx
> ```
> 
> 对应的github actions也同步更新了表单参数，github actions表单默认输出格式是deb。 ![image](https://user-images.githubusercontent.com/28218658/226083809-fb78a43e-16ac-4d1c-9c2f-01d17adb06ae.png)

搞定了，谢谢

**Reply by Tlntin** [reply]
> 搞定了，谢谢

不客气
